src/lib/helpers.ts

```diff
@@ -93,7 +93,7 @@ export function safeParseQueryString<T>(
     try {
       const castValue = params.schema.cast(parsed);
       return params.schema.isValidSync(castValue)
-        ? (castValue as unknown as T) ?? null
+        ? ((castValue as unknown) as T) ?? null
         : null;
```

Comment:
Were these automated changes or did you make these? Not entirely sure what they are used for but just checking they are suppose to be included

Reply:
Reverted it back 👍